code/game/objects/structures/bounty_post.dm

```diff
@@ -0,0 +1,268 @@
+#define BOUNTY_COOLDOWN 10 MINUTES
+#define BOUNTY_MARKER "bounty marker"
+#define EASY_BOUNTY_MULTIPLIER 1.25
+#define MEDIUM_BOUNTY_MULTIPLIER 1.5
+#define HARD_BOUNTY_MULTIPLIER 2
+
+GLOBAL_LIST_EMPTY(holy_contracts)
+GLOBAL_LIST_EMPTY(contract_points)
+GLOBAL_LIST_EMPTY(assigned_bounties)
+GLOBAL_LIST_EMPTY(bounty_points_tracker)
+
+GLOBAL_LIST_INIT(possible_monsters, list(
```

Comment:
This should probably be named something like `possible_holy_contract_target_types` or something, `possible_monsters` could mean a lot